Mlem/API/APIClient/APIClient.swift

```diff
@@ -161,8 +161,6 @@ class APIClient {
         
         if let overrideToken {
             urlRequest.setValue("Bearer \(overrideToken)", forHTTPHeaderField: "Authorization")
-        } else if case let .authenticated(_, token) = session {
-            urlRequest.setValue("Bearer \(token)", forHTTPHeaderField: "Authorization")
```

Comment:
This was breaking the `GetSiteRequest(instanceURL: )` initialiser. I don't think we need this?